jcl/source/windows/JclDebug.pas

```diff
@@ -4187,6 +4382,16 @@ function TJclDebugInfoTD32.GetLocationInfo(const Addr: Pointer; out Info: TJclLo
     end;
 end;
 
+function TJclDebugInfoTD32.GetAddress(const UnitName : string; const ProcName : String): Pointer;
+var
+  VA : Cardinal;
+begin
+  result := nil;
+  VA := FImage.TD32Scanner.VAFromUnitAndProcName( UnitName, ProcName );
+  if ( VA <> 0 ) then
+    result := AddrFromVA( VA );
+end;
+
```

Comment:
Same as above. Note that it's the third occurrence of the exact same code, what's the point of having made this method `virtual abstract` to have it overriden the same in all places?